src/content/blog/2024/nextflow-24.04-highlights.md

```diff
@@ -156,10 +156,10 @@ workflow {
   Channel.topic('versions') // [!code ++]
     | unique()
     | map { process, name, version ->
-      """
+      """\
```

Comment:
I don't think that we need this do we?

Reply:
Or rather, I should ask: was it intentional to add this and if so why?

Reply:
You are right, we do not necessarily need the backslash here!
What it does is that it prevent Groovy from inserting a newline immediately after opening the triple quotes.

So this would be the output _with the backslash_:
```
PREPROCESSING:
  cutadapt: 4.7
PREPROCESSING:
  ribodetector: 0.3.1
GET_SAMPLE_INFO:
  scd: 1.4.22
```
... and this would be the output _without the backslash_:
```

PREPROCESSING:
  ribodetector: 0.3.1

PREPROCESSING:
  cutadapt: 4.7

GET_SAMPLE_INFO:
  scd: 1.4.22
```

Saw the backslash in Ben Sherman's slack reply, though it was cool way of removing extra newlines :)  But the pipeline would run either way, so the backslash in this case is a preference thing, I guess. 

<img width="734" alt="image" src="https://github.com/user-attachments/assets/d0dd03fd-c66e-494d-8d36-c4ec387dd738">

Reply:
Ben also mentioned `.leftTrim()`, this would be the output (looks the same way as when adding the backslash):
```
PREPROCESSING:
  cutadapt: 4.7
PREPROCESSING:
  ribodetector: 0.3.1
GET_SAMPLE_INFO:
  scd: 1.4.22
```

Reply:
Ah ok great, let's leave it in then! I was worried that it would kill the newline and break the yaml, but if it's double spaced already then that's perfect.

Thanks for the explanation! 🙌🏻